contrib/kube-prometheus/README.md

```diff
@@ -279,6 +281,7 @@ These are the available fields with their respective default values:
         names: 'k8s',
         replicas: 2,
         rules: {},
+        enableAdminAPI: 'false',
```

Comment:
this is generated, and since we removed it from the `example.jsonnet`, this should be removed here as well